loadable_apps/loadable_sample/wifiapp/Makefile

```diff
@@ -62,6 +62,9 @@ endif
 ifneq ($(filter y,$(CONFIG_EXAMPLES_BINARY_UPDATE_TEST) $(CONFIG_EXAMPLES_UPDATE_AGING_TEST)),)
 SRCS += binary_update.c
 endif
+ifeq ($(CONFIG_WIFI_PLATFORM_LIB),y)
+LIBSRCS += wifi_pf.c
```

Comment:
I think `wifi_pflib.c` or `wifi_pf_lib.c` is better.

Reply:
I named it wifi_pf because it can be part of library or main wifi binary. 
Just wanted to differentiate between wifi app code and pf code. 
Pls let me know what you think.

Reply:
Okay. I agree your opinion. Let's keep it.